Libraries/Animated/src/AnimatedImplementation.js

```diff
@@ -2142,6 +2245,55 @@ var stagger = function(
   }));
 };
 
+type LoopAnimationConfig = AnimationConfig & { iterations: number };
+
+var loop = function(
+  animation: CompositeAnimation,
+  config: LoopAnimationConfig,
+): CompositeAnimation {
+  if (!config) config = {}; // Use empty config to avoid checking if (config && ...) every time
+  if (!config.iterations) config.iterations = -1; // Loop indefinitely if no 'iterations' provided
```

Comment:
We probably still want to be able to pass 0 for iterations, this can be fix as per my previous comment.